component/src/main/resources/installation-scripts/install.sh

```diff
@@ -31,28 +31,38 @@ ballerina_lib_location=$ballerina_home/bre/lib
 ballerina_balo_location=$ballerina_home/lib/repo
 version=${project.version}
 module_name=kafka
+fileNamePattern="wso2-kafka-module-*.*.*.jar"
```

Comment:
Shall we use `wso2-kafka-*` here, since there might be users who installed older versions of Kafka connector, which named as `wso2-kafka-package-*` instead of `wso2-kafka-module` ?